librz/util/print.c

```diff
@@ -1518,10 +1526,46 @@ static bool check_arg_name(RzPrint *print, char *p, ut64 func_addr) {
 	return false;
 }
 
-static bool ishexprefix(char *p) {
+static inline bool is_hex_prefix(const char *p) {
 	return (p[0] == '0' && p[1] == 'x');
 }
 
+static inline bool is_hex_postfix(const char c) {
```

Comment:
Please be careful to also properly handle UTF-8 Unicode output, at least maybe a couple unit tests could be helpful here.